service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -742,8 +751,20 @@ private boolean dontSendLinkToFondyIf(int sumToPay, Certificate certificate, Ord
         return false;
     }
 
+    private void checkAmountOfBagsIfCourierLimitByAmountOfBag(CourierLocations courierLocations,
+        Integer countOfBigBag) {
+        if (courierLocations.getCourierLimit().equals(CourierLimit.LIMIT_BY_AMOUNT_OF_BAG)) {
```

Comment:
to avoid NPE, rewrite to 
`CourierLimit.LIMIT_BY_AMOUNT_OF_BAG.equals(courierLocations.getCourierLimit())`.